src/app/button/button.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.scss']
+})
+export class ButtonComponent implements OnInit {
+  @Input() text: any;
+  constructor() {}
```

Comment:
Could you add other `(input)`attributes to customize the styling of the button:
examples:
- elevation (drop-shadow)
- background-color
- text color
- border-radius
- disabled status
-